packages/wallet/src/views/Home.jsx

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright (c) Kernel
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { ethers } from 'ethers'
+import { useEffect, useState } from 'react'
+import { linesVector } from '@kernel/common'
+
+import Page from 'components/Page'
+
+const WALLET_STORE_VERSION = '1'
+
+const getItem = (k) => JSON.parse(localStorage.getItem(k))
+
+const loadWallet = () => {
+  const wallet = getItem('wallet')
+  if (wallet.version !== WALLET_STORE_VERSION) {
+    throw new Error('unsupported wallet')
+  }
+  return wallet
+}
+
+const everyoneCards = [
+  {
+    title: 'Claim',
+    description: 'Get set up with some test ETH',
+    url: '/claim',
+    active: false
+  },
+  {
+    title: 'Transact',
+    description: 'Understand how transactions work',
+    url: '/transact',
+    active: false
+  },
+  {
+    title: 'Deploy',
+    description: 'Create your own token now',
+    url: '/deploy',
+    active: false
+  },
+  {
+    title: 'Mint',
+    description: 'Create an NFT in one go',
+    url: '/mint',
+    active: false
+  },
+  {
+    title: 'Contact',
+    description: 'Keep your friends\' addresses safe',
+    url: '/contact',
+    active: false
+  },
+  {
+    title: 'Contribute',
+    description: 'Learn how to improve this with us',
+    url: '/contribute',
+    active: false
+  },
+  {
+    title: 'Interact',
+    description: 'Explore all the best contracts',
+    url: '/interact',
+    active: false
+  },
+  {
+    title: 'Develop',
+    description: 'Write your own code!',
+    url: '/develop',
+    active: false
+  }
+]
+
+const kernelCards = [
+  {
+    title: 'UnProfile',
+    description: 'Unprofile yourself',
+    url: 'https://staging.unprofile.kernel.community/',
```

Comment:
This url is going to cause issues when deploying to a different environment other than staging.

There is an environment variable `REACT_APP_DEPLOY_TARGET` that will tell you what the target environment is.

We should make the URLs dependent on the target.

There's an open issue to update the footer to include links to the apps and the code should be shared across both use cases. https://github.com/kernel-community/services/issues/19

Reply:
@andytudhope you may beat Alon to this, so if you want to use the guidance in #19 ("How to get the correct domains") to create a helper that returns the correct url for each app, go ahead. just let him know so that he can reuse it for the footer.